docker/ddns/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
As the script relies on `bash`, and may fail in a simpler POSIX plain `sh` (or even `bash` in POSIX mode), it is advisable to set it as `#!/bin/bash`